controllers/update_status.go

```diff
@@ -448,7 +449,8 @@ func CheckAndSetProcessStatus(r *FoundationDBClusterReconciler, cluster *fdbtype
 			versionMatch = process.Version == cluster.Spec.Version || process.Version == fmt.Sprintf("%s-PRERELEASE", cluster.Spec.Version)
 		}
 
-		correct = correct && versionMatch
+		// If the `EmptyMonitorConf` is set, the commandline is by definition wrong since there should be no running processes.
+		correct = correct && versionMatch && !cluster.Spec.Buggify.EmptyMonitorConf
```

Comment:
Shouldn't that be rather an `||` to set the `correct` to true if we set `EmptyMonitorConf` otherwise all processes will have the `IncorrectCommandLine` but since we explicitly don't want yo have any running process that shouldn't set the condition.

Reply:
I have to try that in my dev environment. My expectation would be that once we killed the processes (and they are not running anymore) we don't want to have the `IncorrectCommandLine` condition anymore since we reached the desired state. If we still set the `IncorrectCommandLine` even if all processes are shutdown that might be confusing for users without additional knowledge.

Reply:
I'm certainly open to suggestions. I don't think there is a well resolved state for the operator when every fdbserver process is shut down. I also don't expect this to be used in normal operation.

Reply:
I think it's fine for buggify conditions to leave the cluster in an unreconcilable state, as long as we can change the options back after that and get the cluster back to a good state.